packages/home/config.toml

```diff
@@ -1,48 +1,80 @@
-    "Locking for 12 cycles (even beyond block #700000):", "[**SP700C57YJFD5RGHK0GN46478WBAM2KG3A4MN2QJ**](https://explorer.stacks.co/address/SP700C57YJFD5RGHK0GN46478WBAM2KG3A4MN2QJ)",
-    "The more cycles you choose, the less cool down cycles you have. After each locking period, there is one cycle where you can't receive any rewards on your address. You can create a new address and delegate from there.",
+    ]
+    wallet = "https://www.hiro.so/wallet"
+
+# Pool Section
+[params.pool]
+    enable = true
+
+    # Text before table
+    prefix = ["You can use one of the Friedger Pool delegatee addresses here: **Do not send any Stacks token to this address, follow the delegated stacking flow in your preferred wallet**"]
+    
+    # Text after table
+    suffix = ["The more cycles you choose, the less cool down cycles you have. After each locking period, there is one cycle where you can't receive any rewards on your address. You can create a new address and delegate from there.",
     "Please do not stack less than 40 STX. Small payouts are hard to manage.",
     "We start locking your STX around 500 blocks before the start of the next cycle. You can join until 200 blocks before the start. The pool will be closed around 50 blocks before the start.",
     "Also, Friedger Pool will re-stack your STXs for the next cycles without any action from you when you chose to delegate for indefinite duration. If you want to leave the pool, just revoke in Stacks Wallet.",
     "You can track Bitcoin earnings here, which gets converted to STX and paid to you within 2 days after the last reward was received:",
-    "Bitcoin address:", "[**33WSGLeVoEpuZDjB54HKZ1y5YsERELoVNq**](https://www.blockchain.com/btc/address/33WSGLeVoEpuZDjB54HKZ1y5YsERELoVNq)",
-    "The deprecated address [SPCKCVQ6FTZJQYJ42FGE0FCJ1Y1QMTAK0P2WMGX7](https://explorer.stacks.co/address/SPCKCVQ6FTZJQYJ42FGE0FCJ1Y1QMTAK0P2WMGX7?chain=mainnet) has reached end of use and Stacks on this address won't be re-stacked. Also the old Bitcoin address [**1AJm7XWhbPTFqgtJT6oS9wQ3XThk7rA2yF**](https://www.blockchain.com/btc/address/1AJm7XWhbPTFqgtJT6oS9wQ3XThk7rA2yF) won't receive any rewards anymore.",
+    "Bitcoin address (new):", "[**33WSGLeVoEpuZDjB54HKZ1y5YsERELoVNq**](https://www.blockchain.com/btc/address/33WSGLeVoEpuZDjB54HKZ1y5YsERELoVNq)",
+    "The deprecated address [SPCKCVQ6FTZJQYJ42FGE0FCJ1Y1QMTAK0P2WMGX7](https://explorer.stacks.co/address/SPCKCVQ6FTZJQYJ42FGE0FCJ1Y1QMTAK0P2WMGX7?chain=mainnet) is still maintained, also the old Bitcoin address [**1AJm7XWhbPTFqgtJT6oS9wQ3XThk7rA2yF**](https://www.blockchain.com/btc/address/1AJm7XWhbPTFqgtJT6oS9wQ3XThk7rA2yF)",
     "More FAQs about delegated stacking at [stacks101.com](https://stacks101.com/stx-stacking/delegation)."]
-    resume = "https://www.hiro.so/wallet"
+
+    # Display Pool addresses here   
+    [[params.pool.row]]
```

Comment:
The pool address table should be higher up, or at least there should be a clear button how to get to the instructions with addresses.

Reply:
Again, separate page for the Pool. What do you think?

Reply:
new Pool page in next PR